packages/e2e/features/ui/components/authenticator/form-fields.feature

```diff
@@ -55,9 +55,10 @@ Feature: Form Fields
     Then "Password" field does not have class "amplify-visually-hidden"
     Then I see placeholder "Enter your Password:"
 
+  @angular @react @vue
```

Comment:
This is the only change here -- not sure why this skipped before so I re-enabled it with updated field values.

Reply:
To fasten our tests 😈